files/docker/node/dockerfile_bin

```diff
@@ -55,11 +56,11 @@ RUN set -x && apt update \
     && apt-get -y autoremove \
     && rm -rf /var/lib/apt/lists/*
 
-
 RUN set -x && export SUDO='N' \
     && export UPDATE_CHECK='N' \
     && export SKIP_DBSYNC_DOWNLOAD='Y' \
-    && ./guild-deploy.sh -b ${GUILD_DEPLOY_BRANCH} -s dcmowx \
+    && export G_ACCOUNT=${G_ACCOUNT} \
```

Comment:
Ditto.

Reply:
ditto is very low effort. please communicate more effectivly